### PR TITLE
Redesigning the homepage!

### PR DESCRIPTION
**Pull request by bahorn**
Need feedback on the usual things, but should be much more engaging!

### CODE REVIEW

**Comment by bahorn** [comment]
Seems this hides the table of contents on the homepage. So no longer linking to the art page, etc.

**Comment by bahorn** [comment]
Feedback so far:

* @robzwolf mentioned it'd be good if the colors on the buttons had better contrast when hovered over.

File: docs/events/card.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+
```
Comment (jmsv) [inline docs/events/card.md:1]: why so many space at start of file

File: docs/events/list.md [hunk 2]
```diff
@@ -1,143 +1,51 @@
 # Upcoming Events
 This list showcases upcoming UK student-run hackathons and hackathon-associated events.
 
-Want to add something to this list? [Fork and pull request](https://github.com/Hackathons-UK/wiki/edit/master/docs/index.md) to add your event!
+Want to add something to this list? [Fork and pull request](https://github.com/Hackathons-UK/wiki/edit/master/mkdocs.yml) to add your event! You can find the hackathons inside <code>hackathon_seasons:</code>
```
Comment (jmsv) [inline docs/events/list.md:4]: can we use markdown code syntax (backticks) rather than html `code` tags?

File: docs/index.md [hunk 3]
```diff
@@ -1,51 +1,28 @@
...
-|-----------------|-------|--------------------|--------------|----|
-| HackTheMidlands 5.0 | [hackthemidlands.com](https://hackthemidlands.com) | Birmingham (Venue TBC) | 300 | 24th-25th October 2020 (TBC) |
-| R.U. Hacking? 2020 | [ruhacking.me](https://ruhacking.me) | Reading University | TBC | 24th-25th October 2020 |
-| Hack Brunel 2.0  | [hackbrunel.com](https://hackbrunel.com/) | Brunel University London | 150 | 31st October-1st November 2020 |
-| hackSheffield 6.0 | [hacksheffield.com](https://hacksheffield.com) | University of Sheffield | 200 | 7th-8th November 2020 |
-| DurHack 2020    | [durhack.com](https://durhack.com) | Durham University | 425 | 14th-15th November 2020 |
-| HackNotts 2020  | [hacknotts.com](https://www.hacknotts.com) | University of Nottingham | 200 | 28th-29th November 2020 |
+{% endfor %}
 
+Want to add something to this list? [Fork and pull request](https://github.com/Hackathons-UK/wiki/edit/master/mkdocs.yml) to add your event! You can find the hackathons inside <code>hackathon_seasons:</code>
```
Comment (jmsv) [inline docs/index.md:25]: Is this supposed to link to `mkdocs` file or just the repo?

Comment (bahorn) [reply docs/index.md:25]: Should be the repo

File: docs/events/list.md (hunk 2 above)
Comment (Tello-Wharton) [reply docs/events/list.md:4]: Sounds good, will sort!

File: docs/index.md (hunk 3 above)
Comment (Tello-Wharton) [reply docs/index.md:25]: The mkdocks

Comment (jmsv) [reply docs/index.md:25]: Why mkdocs file? If we're inviting people to contribute a link to the repo makes more sense imo

Comment (Tello-Wharton) [reply docs/index.md:25]: If you wanted the repo am cool with that... but previously it linked to the file that had the hackathon data!

Comment (Tello-Wharton) [reply docs/index.md:25]: The file has changed, that's all

**Comment by bahorn** [comment]
Think remaining thing i want to get done is splitting the events list from the main yaml.

surely mkdocs must have an import feature!

**Comment by Tello-Wharton** [comment]
> Think remaining thing i want to get done is splitting the events list from the main yaml.
> 
> surely mkdocs must have an import feature!

Agree. You need this do be done before we get this out there?

**Comment by bahorn** [comment]
Seems the best way of doing that will be to implement a new mkdocs plugin, so beyond scope of this PR.

File: docs/index.md (hunk 3 above)
Comment (jmsv) [reply docs/index.md]: > If you wanted the repo am cool with that... but previously it linked to the file that had the hackathon data!

ahhh got it, makes sense

File: docs/events/card.md (hunk 1 above)
Comment (bahorn) [reply docs/events/card.md]: resolved

**Comment by robzwolf** [comment]
This is absolutely amazing! Kudos to everyone involved, the new design brings a fresh coat of paint to the wiki. 😁 

I’ve opened an issue #206 about a small styling issue that I don’t recall being there before, but not sure if it’s related to this PR. 

And again – this PR is awesome, great job by everyone involved 😎

**Pull request merged**